app/Http/Controllers/FontsController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Http;
+
+class FontsController extends Controller
+{
+    public function index(Request $request)
+    {
+        return \Cache::remember('google_fonts', 60 * 60, function () {
+            $url = "https://www.googleapis.com/webfonts/v1/webfonts?key=" . config('services.google_fonts_api_key');
+            $response = Http::get($url);
+            if ($response->successful()) {
+                $fonts = collect($response->json()['items'])->map(function ($font) {
```

Comment:
Let's filter out iconset (they probably have a specific attribute you can use to filter out - either in the api endpoint or in the back-end here)

![CleanShot 2024-08-13 at 11 02 51@2x](https://github.com/user-attachments/assets/f5f3e14e-6274-4684-ae11-f006dc8e9cca)